src/node_constants.cc

```diff
@@ -800,6 +800,15 @@ void DefineCryptoConstants(Local<Object> target) {
     NODE_DEFINE_CONSTANT(target, OPENSSL_VERSION_NUMBER);
 #endif
 
+    // When defined, its defined without a value, so redefine it as 1 or 0.
+#ifdef OPENSSL_NO_SSL_TRACE
+# undef OPENSSL_NO_SSL_TRACE
+# define OPENSSL_NO_SSL_TRACE 1
+#else
+#  define OPENSSL_NO_SSL_TRACE 0
+#endif
+    NODE_DEFINE_CONSTANT(target, OPENSSL_NO_SSL_TRACE);
```

Comment:
This should be documented somewhere?

Reply:
Constants aren't documented unless they are expected to be used in our API, this isn't.